src/components/app/ErrorBoundary.css

```diff
@@ -0,0 +1,41 @@
+.appErrorBoundary {
+  padding: 10px;
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: var(--grey-20);
+  overflow: hidden;
+  box-sizing: border-box;
```

Comment:
can you put `box-sizing` towards the top? Actually, is it even needed here, given we don't define the width or height?